cmd/listservices.go

```diff
@@ -0,0 +1,68 @@
+// Copyright © 2020 Developer <developer@vamp.io>
+//
+// Permission is hereby granted, free of charge, to any person obtaining a copy
+// of this software and associated documentation files (the "Software"), to deal
+// in the Software without restriction, including without limitation the rights
+// to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+// copies of the Software, and to permit persons to whom the Software is
+// furnished to do so, subject to the following conditions:
+//
+// The above copyright notice and this permission notice shall be included in
+// all copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+// THE SOFTWARE.
+
+package cmd
+
+import (
+	"fmt"
+
+	"github.com/magneticio/forklift/core"
+	"github.com/magneticio/forklift/logging"
+	"github.com/spf13/cobra"
+	yaml "gopkg.in/yaml.v3"
+)
+
+var listServicesCmd = &cobra.Command{
+	Use:   "services",
+	Short: "List existing services",
+	Long: AddAppName(`List existing services
+    Usage:
+    $AppName list services --cluster <cluster_id> --application <application_id>`),
```

Comment:
I might be missing something but cluster_id doesn't seem to be used

Reply:
`cluster` and `project` values can be configured in config file, env variables or using persistent flags (see here: `cmd/root.go`). That's why you cannot see them directly added to subcommands. Cluster id is then used in ListServices function in `core/core.go`.